--- libsxg.orig/CMakeLists.txt	2020-01-22 05:09:03.920824590 +0000
+++ libsxg/CMakeLists.txt	2020-01-22 05:09:12.212572092 +0000
@@ -20,6 +20,9 @@
 set (CMAKE_PROJECT_VERSION_MINOR "2")
 set (CMAKE_PROJECT_VERSION_PATCH "0")
 
+set (LIBSXG_CORE_VERSION "${CMAKE_PROJECT_VERSION_MAJOR}.${CMAKE_PROJECT_VERSION_MINOR}")
+set (LIBSXG_SOVERSION "${CMAKE_PROJECT_VERSION_MAJOR}")
+
 option (RUN_TEST
   "If false, build libsxg without building tests.\
   Building tests of libsxg requires more recent version of cmake (>= 3.13).\
@@ -126,8 +129,7 @@
 
 set_target_properties(
   sxg
-  PROPERTIES VERSION ${CMAKE_PROJECT_VERSION}
-  SOVERSION "${CMAKE_PROJECT_VERSION_MAJOR}.${CMAKE_PROJECT_VERSION_MINOR}"
+  PROPERTIES VERSION ${LIBSXG_CORE_VERSION} SOVERSION ${LIBSXG_SOVERSION}
 )
 
 set_target_properties (sxg PROPERTIES PUBLIC_HEADER "${HEADERS}")
